dev/packages/example/cisco-1.0.0/dataset/asa/agent/input/input.yml

```diff
@@ -0,0 +1,14 @@
+{{#if input == "syslog"}}
```

Comment:
My comment further down in the filebeat config should have gone here.

Also if I remember correctly, the directly should be called stream/stream.yml